backend/utils/walk.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+const walk = function(dir, done) {
```

Comment:
Very bad implementation. I would recommend to use Promise version of fs methods and with using async/await code will be much cleaner

Reply:
same here